mechanicalsoup/browser.py

```diff
@@ -38,7 +38,7 @@ def post(self, *args, **kwargs):
         return response
 
     def _build_request(self, form, url=None, **kwargs):
-        method = form.get("method", "get")
+        method = str(form.get("method", "get"))
```

Comment:
We expect method will always be "get" or "post" in practice, so no risk of UnicodeEncodeError in Python 2.